### ISSUE TITLE
'select $path from traverse' return null

### ISSUE THREAD

**Issue by blueAnger** [orientechnologies/orientdb#7962]
### OS: <CentOS 7>  

according to this [documentation](http://orientdb.com/docs/3.0.x/sql/SQL-Traverse.html), one can use $path, $stack when traverse. So I run a SQL like

```
select $path from (traverse both() from #153:28622 maxdepth 2 strategy breadth_first)
```

The result is a list and all elements within are nulls.  But the result of the 'traverse' sub-command is not empty. I tried the SQL above through orientdb-self-provided web page as well as ODatabaseSession.query(), it just won't work. 

FYI, according to my tests, the only context variable that works is $depth.

**Comment by luigidellaquila** [comment]
Hi @blueAnger 

It seems to be a regression, I'm checking it now

Thanks

Luigi

**Comment by luigidellaquila** [comment]
Hi @blueAnger 

I just fixed it, but it's slightly different from v 2.2. Now, instead of returning a string, the `$path` variable returns a collection (a list) or RIDs, so it's easy to manipulate

Thanks

Luigi
